pkg/operator/bootstrap.go

```diff
@@ -4,14 +4,15 @@ import (
 	"context"
 	"crypto/rand"
 	"fmt"
-
+	"github.com/openshift/cluster-image-registry-operator/pkg/storage/util"
```

Comment:
nit, move this line to other `github.com/openshift/cluster-image-registry-operator` imports

Reply:
fixed, it was there at the beginning I think that gofmt replaced the order for some reason :/